python/arcticdb/arctic.py

```diff
@@ -149,8 +148,13 @@ def __init__(self, uri: str, encoding_version: EncodingVersion = EncodingVersion
         self._library_adapter = _cls(uri, self._encoding_version)
         self._library_manager = LibraryManager(self._library_adapter.config_library)
         self._uri = uri
+        self._open_libraries = dict()
```

Comment:
Should consider managing this with the "EnvironmentConfig.lib_by_path" dict

Reply:
I think that's quite a separate concept no? That should be for tracking all the libs that an environment _knows about_ (but which might not be "opened").